functorch/test/test_vmap.py

```diff
@@ -3222,6 +3222,7 @@ def test():
         xfail('eye', ''),  # non-tensor input
         xfail('broadcast_shapes', ''),  # test runner can't handle non-Tensor ops
         xfail('sparse.sampled_addmm'),  # sparse
+        xfail('cross'),  # The default value of dim in op is *very* weird. No wonder it doesn't work
```

Comment:
for `linalg.cross`, the default is -1. For `cross`... well... it actually has one of the craziest defaults in the whole PyTorch.
> If dim is not given, it defaults to the first dimension found with the size 3. Note that this might be unexpected.